composition-js/src/genHintDoc.ts

```diff
@@ -0,0 +1,88 @@
+import { assert, MultiMap } from '@apollo/federation-internals';
+import { HintCodeDefinition, HintLevel, HINTS } from './hints';
+
+const header = `---
+title: Federation Hints
+sidebar_title: Hints
+---
+
+When Apollo Gateway successfully **compose** the schemas provided by your [subgraphs](./subgraphs/) into a **supergraph schema**, it may generate a number of "hints" that provide additional information the composition.
+
+Hints are categorized under the following levels:
+1. WARN: indicates a situation that may expected but is usually temporary and should be double-checked. Typically, composition might have had to ignore some elements from some subgraph when creating the supergraph.
+2. INFO: information that may hint at some improvements or highlight noteworthy resolution made by composition, but that can otherwise be ignored.
```

Comment:
**nitpick**

```suggestion
2. INFO: information that may hint at some improvements or highlight noteworthy resolution made by composition but can otherwise be ignored.
```